### PR TITLE
REVSDL-2389 OnServiceDeactivated notification implementation

### PR DESCRIPTION
**Pull request by giangtqh**
@bonewell 
@KKarlash 
@VladSemenyuk 
@DKlimenko 
@thinhts 
please review

### CODE REVIEW

File: src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+namespace vr_cooperation {
+
+namespace commands {
+
+class OnServiceDeactivatedNotification {
+ public:
+  /**
+   * @brief OnServiceDeactivatedNotification class constructor
+   * @param parent parent object
+   **/
```
Comment (bonewell) [inline src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h]: `**/` -> `*/`

File: src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+namespace vr_cooperation {
+
+namespace commands {
+
+class OnServiceDeactivatedNotification {
+ public:
+  /**
+   * @brief OnServiceDeactivatedNotification class constructor
+   * @param parent parent object
```
Comment (bonewell) [inline src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h]: Please add details about parent object

File: src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   **/
+  explicit OnServiceDeactivatedNotification(VRModule* parent);
+
+  /**
+   * @brief OnAppPermissionChangedNotification class destructor
+   **/
```
Comment (bonewell) [inline src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h]: This comment is useless, you might delete it.

File: src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  virtual ~OnServiceDeactivatedNotification();
+
+  /**
+   * @brief Execute command
+   * @param message message to be sent to mobile
+   **/
```
Comment (bonewell) [inline src/components/vr_cooperation/include/vr_cooperation/commands/on_service_deactivated_notification.h]: `**/` -> `*/`

File: src/components/vr_cooperation/src/on_service_deactivated_notification.cc [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+ */
+
+#include "vr_cooperation/commands/on_service_deactivated_notification.h"
+#include "utils/logger.h"
+
+namespace vr_cooperation {
+
+namespace commands {
+
+CREATE_LOGGERPTR_GLOBAL(logger_, "OnServiceDeactivatedNotification")
```
Comment (bonewell) [inline src/components/vr_cooperation/src/on_service_deactivated_notification.cc]: Using of component name for every class of that component is better practice.

**Comment by bonewell** [comment]
Reviewed.

**Comment by giangtqh** [comment]
@bonewell 
I fix all your notes, please review

**Comment by bonewell** [comment]
Reviewed.

**Comment by DKlimenko** [comment]
reviewed

**Pull request merged**